build.gradle

```diff
@@ -193,6 +195,9 @@ configurations {
 }
 
 dependencies {
+    // For resource access control
+    compileOnly group: 'org.opensearch', name:'opensearch-security-spi', version:"${opensearch_build}"
```

Comment:
This format is deprecated and will fail in Gradle 10.   Please match the other declarations in this build.

See https://docs.gradle.org/9.2.0/userguide/upgrading_version_9.html#dependency_multi_string_notation